package.json

```diff
@@ -10,7 +10,7 @@
 	"version": "1.0.0",
 	"license": "GPL-2.0-or-later",
 	"engines": {
-		"node": "^22.5.0",
+		"node": ">=20.17.0 <22.5.0",
```

Comment:
@linxpinx I think that's not correct, maybe try to use `">=20.17.0 <23"` ?

Reply:
```suggestion
		"node": "^20.17.0 || ^22.5.0",
```

Reply:
That's better :+1: